src/setops.jl

```diff
@@ -2,13 +2,19 @@
 # Projection operations
 # ========================================
 
-function _project_oa(X::AbstractLazyReachSet, vars, t)
+function _project_oa(X::AbstractLazyReachSet, vars, t; remove_zero_generators=true)
     return Project(X, vars)
 end
 
-function _project_oa(X::AbstractTaylorModelReachSet, vars, t)
-    Z = overapproximate(X, Zonotope, t)
-    return project(Z, vars)
+function _project_oa(X::AbstractTaylorModelReachSet, vars, t; remove_zero_generators=true)
+    Z = overapproximate(X, Zonotope, t, remove_zero_generators=remove_zero_generators)
+
+    # TODO pass option to LazySets
+    πZ = concretize(Projection(set(Z), vars))
+    if remove_zero_generators
+        πZ = LazySets.remove_zero_generators(πZ)
+    end
+    return ReachSet(πZ, tspan(X))
```

Comment:
After mergin https://github.com/JuliaReach/LazySets.jl/pull/2728 we can do:
```suggestion
    return project(Z, vars; remove_zero_generators=remove_zero_generators)
```